### PR TITLE
Czech translation of chapters funcmethodsinterfaces, goteam

### PR DESCRIPTION
**Pull request by klatys**
@michalpristas would you mind looking over these? Thanks in advance

### CODE REVIEW

**Comment by michalpristas** [comment]
sure, will find some time today to go through it, thanks for the translation 👍

File: cs/SUMMARY.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Obsah
 
+## Začínáme s Go
 * [Úvod](README.md)
 * [Začínáme](getting-started/README.md)
 * [Syntaxe a Typy](syntaxtypes/README.md)
-* [Ukazatele a typy reference](pointersreftypes/README.md)
\ No newline at end of file
+* [Ukazatele a typy reference](pointersreftypes/README.md)
+* [Funkce, Methody and Interface (Rozhraní)](funcmethodsinterfaces/README.md)
```
Comment (michalpristas) [inline cs/SUMMARY.md]: s/and/a

File: cs/goteam/README.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+- Jméno balíčku popisuje jeho účel
+- Podle názvu lze jednoduše pochopit co balíček dělá
+- Názvy jsou spíše krátké
+- Pokud je to potřeba, použijeme popisný nadřazený balíček a několik potomků implementujících skutečnou funkcionalitu -- jako u balíčku *encoding*
+
+
+## Organizace balíčků - Aplikace
+
+Všechny balíčky, které jsme viděli jsou knihovny. Předpokládá se, že budou importovány a použity ve spustitelné programu jako je služba nebo nástroj příkazové řádky.
```
Comment (michalpristas) [inline cs/goteam/README.md]: s/spustitelné/spustitelném

File: cs/goteam/README.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+- Externí datové zdroje
+- Logika přenosu dat (http, RPC)
+
+Pro každou závislost by měl existovat vlastní balíček.
+
+Proč samostatný balíček pro každou závislost?
+
+- Jednodušší testování
+- Snadná náhrada/výměna
+- Vyhneme se tak kruhovým závislostem
```
Comment (michalpristas) [inline cs/goteam/README.md:119]: cyklicke vs kruhove, up to you, just a minor thing

File: cs/goteam/README.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+	- var tt []*Thing (var vv []*Věc)
+
+- uvnitř cyklu používejte jednopísmenné proměnné
+	- for i, t := range tt {}
+
+Tyto konvence se běžně používají i v samotném zdrojovém kódu Go.
+
+## Jmenné konvence - funkce a metody
+
+Funkcním v balíčku nepřidávejte do názvu jméno balíčku:
```
Comment (michalpristas) [inline cs/goteam/README.md]: s/Funkcním/Funkcím

File: cs/goteam/README.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+Puritáni by mohli namítnout, že všechny interface by měli mít koncovku `-er`. Dle mého názoru je popisnost a čitelnost přednější.
+
+## Jmenné konvence - Zdrojový kód
+
+Uvnitř balíčku oddělujte kód do logických celků.
+
+Pokud balíček pracuje s více objekty, ponechte logiku pro každý objekt ve vlastním souboru:
+
+	src/goteam/cviceni/sklad/postgres
```
Comment (michalpristas) [inline cs/goteam/README.md]: please, do not translate those paths, they are real paths for code available for download. same goes for filenames

File: cs/goteam/README.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+
+	objednavky.go
+	dodavatele.go
+	produkty.go
+
+V balíčku, který definuje vaše doménové objekty, definujte typy a jejich interface (rozhraní) pro každý objekt v odděleném souboru:
+
+	src/goteam/cviceni/sklad
```
Comment (michalpristas) [inline cs/goteam/README.md]: same here

**Review by michalpristas — CHANGES_REQUESTED** [review]
I'm really bad at procrastination and did review right away

**Comment by klatys** [comment]
Thanks a lot @michalpristas !

**Review by michalpristas — APPROVED** [review]

**Comment by michalpristas** [comment]
@bketelsen Looks good, ready to merge

**Pull request merged**